typescript/formsApps.d.ts

```diff
@@ -332,8 +332,8 @@ type _NewFormsApp = {
   userTokensValidityInMinutes?: number
   /** The id of the Google Maps integration key to be used for Google Maps elements. */
   googleMapsIntegrationKeyId?: string
-  /** List of groups users can be assigned too */
-  groups?: string[]
+  /** List of groups users can be assigned too and email addresses to be notified via */
+  groups?: { name: string[]; notificationEmailAddresses: string[] }
```

Comment:
`name` should just be a single string now yeah? I think you want the `[]` on the object rather than each property